mlflow/models/cli.py

```diff
@@ -132,13 +133,24 @@ def serve(
     help="Specify packages and versions to override the dependencies defined "
     "in the model. Must be a comma-separated string like x==y,z==a.",
 )
+@click.option(
+    "--extra-envs",
+    default=None,
+    type=str,
+    help="Extra environment variables to set when running the model. Must be "
+    'a JSON-formatted string like \'{"key1": "value1", "key2": "value2"}\'.',
+)
```

Comment:
Can you use the same approach as `docker run` where you can pass environment variables like:

```
docker run -e FOO=foo BAR=bar ...
```